Resources/Prototypes/name_identifier_groups.yml

```
@@ -29,6 +29,12 @@
   minValue: 1000
   maxValue: 9999
 
+- type: nameIdentifierGroup
+  id: StationAi
+  prefix: AI
+  minValue: 100
+  maxValue: 999
```

```suggestion
  minValue: 1000
  maxValue: 9999
```
needs to be more like HAL 9000

---

Personally I like it more as 100 to 999 since it makes it more unique in this case, and easier to spot than (SI-1234)
(I think only MMIs use 3 numbers?)

---

I guess it's fine since some default AI names already include the 9000 thing